plugins/contentful-plugin/src/pages/about.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react"
+import { graphql } from "gatsby"
+import Layout from "../components/layout"
+import * as sections from "../components/sections"
+import Fallback from "../components/fallback"
+import SEOHead from "../components/head"
+
+interface AboutProps {
+  data: {
+    aboutPage: {
+      id: string
+      title: string
+      description: string
+      image: { id: string; url: string }
+      blocks: sections.HomepageBlock[]
+    }
+  }
+}
+
+export default function About(props: AboutProps) {
+  const { aboutPage } = props.data
+
+  return (
+    <Layout>
+      {aboutPage ? (
+        aboutPage.blocks.map((block) => {
```

Comment:
I think we should also add something similar to the `src/pages/about.tsx`, just in case somebody runs `yarn start` on root with missing 'about' data